plugins/ui-update/notifications.ts

```diff
@@ -0,0 +1,189 @@
+import * as semver from "semver";
+import { filter } from "rxjs/operators";
+import {
+  ToastAppearance,
+  ToastCallbackType,
+  ToastNotification
+} from "@extension-kid/toast-notifications";
+
+import { getAction$, getUiMetadata$ } from "./streams";
+import { UIActions, UIActionType } from "./types/UIAction";
+import { rollbackUI, updateUI } from "./commands";
+import { getNotificationService } from "./utils";
+import { i18nMark } from "@lingui/react";
+
+const activeNotifications: string[] = [];
+
+function clearActiveNotifications() {
+  if (activeNotifications.length) {
+    activeNotifications.splice(0, activeNotifications.length);
+  }
+}
+
+function addActiveNotification(id: string) {
+  activeNotifications.push(id);
```

Comment:
Should we check if a notification with such an id already exists? Or maybe we can make `activeNotificaitons` a set.

Reply:
I do check if the array includes the `id` in the `displayNotification` function, which was the only function calling this helper. So I just removed this helper function.